packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -316,28 +316,37 @@ void main() {
     });
   });
 
-    testUsingContext('runner disable telemetry with flag', () async {
-      io.setExitFunctionForTests((int exitCode) {});
-
-      expect(globals.analytics.telemetryEnabled, true);
-      expect(globals.analytics.shouldShowMessage, true);
-
-      await runner.run(
-        <String>['--disable-telemetry'],
-        () => <FlutterCommand>[],
-        // This flutterVersion disables crash reporting.
-        flutterVersion: '[user-branch]/',
-        shutdownHooks: ShutdownHooks(),
-      );
+  group('unified_analytics', () {
+    final Usage legacyAnalytics = TestUsage();
```

Comment:
Does this ever get used in the test?

Reply:
Yes, I'm using it in the setup to disable analytics